### DISCUSSION TITLE
Where is this `ctx` coming from?

### DISCUSSION THREAD

**Discussion by tanmaymishu** [uptrace/bun#637] [Q&A] [answered]
Hi, I was going through the documentation here:
https://bun.uptrace.dev/guide/golang-orm.html#connecting-to-a-database

And couldn't find where this `ctx` variable originated. Is it supposed to come from other packages that bun is coupled with?

**Comment by tanmaymishu** [comment, marked as answer]
Found it: https://github.com/uptrace/bun/blob/master/example/basic/main.go#L15

**Reply by thanglequoc** [reply]
I was looking for this as well, thank you @tanmaymishu 😸
